models/service.rb

```diff
@@ -63,6 +63,7 @@ class Service < Base
 
     def self.create(hash, _options = {})
       hash[:skills] = [] if hash[:skills].to_a.empty?
+      hash[:skills].map!(&:to_sym)
```

Comment:
It looks like there are quite a few other fields/parameters which have symbol type in `vrp_input.rb` and because of as_json/to_json they will loose their type after dump/load :/ 

Can you do a similar `:to_sym` call for these fields in this commit please -- then we can rename the commit to something like "Prevent as_json transforming symbols into strings"? 

Hopefully there will not be more members in this situation but it looks like we might want automatize this à-la-façon-de `as_json: :id` in base.rb -- like `type: symbol` that way the call to `:to_sym` would be done automatically in `initialize()`.

Reply:
There is apparently a few `type: Date` and  `type: DateTime` as well. And we need to keep in mind "solution" side of things -- for example,  there is `field :schedule_start_date` in `solution_configuration.rb` which comes from the original `Configuration::Schedule` so we need to apply the `type: Date` to both of them